realwork/realwork/ArtistMediaTableViewController.swift

```diff
@@ -33,10 +36,24 @@ class ArtistMediaTableViewController: UITableViewController {
     }
 
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCell(withIdentifier: "Cell", for: indexPath)
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: "Cell", for: indexPath) as? TableCell else {
+            return UITableViewCell()
+        }
 
         // Configure the cell...
-        cell.textLabel?.text = listOfArtistCollections[indexPath.row].artistName
+        cell.nameLbl.text = listOfArtistCollections[indexPath.row].artistName
+        cell.titleLbl.text = listOfArtistCollections[indexPath.row].collectionName
+        guard let imageURL = URL(string: listOfArtistCollections[indexPath.row].artworkUrl60) else {
+            return UITableViewCell()
+        }
```

Comment:
This guard statement would cause a default UITableViewCell if no image URL could be found. Is this intended? If not then I suggest an if let statement